roles/umount_remove_lv/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+- name: 'Umount requested lv'
+  mount:
+    path: '{{ item }}'
+    state: absent
+  with_items: "{{ mountpoint_list|map(attribute='mount')|list }}"
```

Comment:
We'll likely also want to remove the mount directory under `/mnt/...`.

Reply:
'absent' specifies that the device mount's entry will be removed from fstab and will also unmount the device and remove the mount point. (https://docs.ansible.com/ansible/2.5/modules/mount_module.html)